app/assets/stylesheets/application.scss

```
@@ -1,13 +1,15 @@
-@import "autocomplete";
-@import "hods-loading-spinner";
+@use "autocomplete";
+@use "hods-loading-spinner";
+@use "partials/heading";
+@use "partials/legend";
 
 .js-visible {
   display: none !important;
 }
 .js-enabled .js-visible {
   display: inline-block !important;
 }
-.long-word, .govuk-error-summary, .govuk-error-message {
+.govuk-error-summary, .govuk-error-message {
   word-wrap: break-word;
```

```suggestion
  overflow-wrap: break-word;
```

I think the correct property to use is `overflow-wrap`.

Mozilla's [Wrapping and breaking text](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text) mentions:

> The overflow-wrap property acts in the same way as the non-standard property word-wrap. The word-wrap property is now treated by browsers as an alias of the standard property.

In the [govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/8f8f7bef9bac077b5d3af0cfe7a9fe586531ae22/packages/govuk-frontend/src/govuk/components/panel/_index.scss#L30) I found that they use the property `overflow-wrap` and also use `word-wrap: break-word;` to support IE.

So we could use:
```suggestion
  overflow-wrap: break-word;
  
  // Support IE
  word-wrap: break-word;
```